app/scripts/metamask-controller.js

```diff
@@ -2922,14 +2929,22 @@ export default class MetamaskController extends EventEmitter {
               origin,
               args.message,
             ),
-          showInAppNotification: (origin, args) =>
+          showInAppNotification: (origin, args) => {
+            const { message, title, footerLink } = args;
+            const notificationArgs = {
+              interfaceId: args.content,
+              message,
+              title,
+              footerLink,
+            };
             this.controllerMessenger.call(
```

Comment:
We need to return the response here otherwise every call to `snap_notify`  will throw
```
Request for method \"snap_notify\" returned undefined result."
```